java/src/main/java/org/eclipse/ditto/client/management/CommonManagement.java

```diff
@@ -181,6 +198,42 @@
      */
     CompletableFuture<Thing> create(JsonObject thing, Option<?>... options);
 
+    /**
+     * Creates an empty {@link Thing} for the given identifier.
+     *
+     * @param thingId the identifier of the Thing to be created. It must conform to the namespaced
+     * entity ID notation (see Ditto documentation).
+     * @param initialPolicy a custom policy to use for the Thing instead of the default Policy.
+     * @param options options to be applied configuring behaviour of this method, see {@link
+     * org.eclipse.ditto.client.options.Options}.
+     * @return completable future providing the created Thing object or a specific {@link
+     * org.eclipse.ditto.model.base.exceptions.DittoRuntimeException} if the operation failed
+     * @throws IllegalArgumentException if {@code thingId} is {@code null} or empty.
+     * @throws org.eclipse.ditto.model.things.ThingIdInvalidException if the {@code thingId} was invalid.
+     * @since 1.1.0
+     */
+    CompletableFuture<Thing> create(ThingId thingId, JsonObject initialPolicy, Option<?>... options);
+
+    /**
+     * Creates a {@link Thing} based on the given {@link JsonObject}.
+     *
+     * @param thing a JSON object representation of the Thing to be created. The provided JSON object is required to
+     * contain a field named {@code "thingId"} of the basic JSON type String which contains the identifier of the Thing
+     * to be created. It must conform to the namespaced entity ID notation (see Ditto documentation).
+     * @param initialPolicy a custom policy to use for the Thing instead of the default Policy.
+     * @param options options to be applied configuring behaviour of this method, see {@link
+     * org.eclipse.ditto.client.options.Options}.
+     * @return completable future providing the created Thing object or a specific {@link
+     * org.eclipse.ditto.model.base.exceptions.DittoRuntimeException} if the operation failed
+     * @throws IllegalArgumentException if {@code thing} is {@code null} or if it does not contain the field named
+     * {@code "thingId"}.
+     * @throws org.eclipse.ditto.model.base.exceptions.DittoJsonException if {@code thing} cannot be parsed to a {@link
+     * Thing}.
+     * @throws org.eclipse.ditto.model.things.ThingIdInvalidException if the {@code thingId} was invalid.
+     * @since 1.1.0
+     */
+    CompletableFuture<Thing> create(JsonObject thing, JsonObject initialPolicy, Option<?>... options);
```

Comment:
Is there a particular reason why the create and put methods with `initialPolicy` all only provide `JsonObject` instead of a `Policy` object?
That would be way more convenient to use IMO. Either in addition to the `JsonObject` or maybe even replacing it.

Also some have the `initialPolicy` param `@Nullable`, others don't. Is that intentionally?

Reply:
Providing create and put methods with `initialPolicy` using a `Policy` object is a good point. I include this in addition to the existing possibilities.

The methods with the `@Nullable` once in put and in the create method are called by all other put and create methods. This is where the actual processing takes place. 
Maybe it would be better to introduce a private method which does the processing? This would remove the `@Nullable` from the two public put and create methods.